drivers/i2c/i2c_gecko.c

```diff
@@ -198,7 +197,6 @@ static const struct i2c_driver_api i2c_gecko_driver_api = {
 static struct i2c_gecko_config i2c_gecko_config_0 = {
```

Comment:
This could be made `const` and so put into flash, since `struct device` assumes it'll be in flash.  Same for the other instances.

Reply:
done